doc/intro.md

```diff
@@ -311,18 +300,18 @@ The same holds true for blocks themselves once we realize a block is simply a se
 
     sum(outputs) - sum(inputs) = sum(kernel_excess)
 
-Simplifying slightly, (again ignoring transaction fees) we can say that Mimblewimble blocks can be treated exactly as Mimblewimble transactions.
+Simplifying slightly (again ignoring transaction fees), we can say that Mimblewimble blocks can be treated exactly as Mimblewimble transactions.
 
 ##### Kernel Offsets
 
-There is a subtle problem with Mimblewimble blocks and transactions as described above. It is possible (and in some cases trivial) to reconstruct the constituent transactions in a block. This is clearly bad for privacy. This is the "subset" problem: given a set of inputs, outputs, and transaction kernels a subset of these will recombine to reconstruct a valid transaction.
+There is a subtle problem with Mimblewimble blocks and transactions as described above. It is possible (and in some cases trivial) to reconstruct the constituent transactions in a block. This is clearly bad for privacy. This is the "subset" problem: given a set of inputs, outputs and transaction kernels, a subset of these will recombine to reconstruct a valid transaction.
 
 Consider the following two transactions:
 
     (in1, in2) -> (out1), (kern1)
     (in3) -> (out2), (kern2)
 
-We can aggregate them into the following block (or aggregate transaction):
+We can aggregate them into the following block (or aggregated transaction):
```

Comment:
The term "aggregate transaction" is commonly used across the docs and the codebase.
We should retain this here.